frame/collective/src/lib.rs

```diff
@@ -361,19 +361,11 @@ pub mod pallet {
 		/// Any call to `set_members` must be careful that the member set doesn't get out of sync
 		/// with other logic managing the member set.
 		///
-		/// # <weight>
-		/// ## Weight
+		/// ## Complexity:
 		/// - `O(MP + N)` where:
 		///   - `M` old-members-count (code- and governance-bounded)
 		///   - `N` new-members-count (code- and governance-bounded)
 		///   - `P` proposals-count (code-bounded)
-		/// - DB:
```

Comment:
Huh, why is this also being removed?

Reply:
DB information is now generated automatically by the benchmarking process in the weights file.